code/modules/mob/living/carbon/hallucination.dm

```diff
@@ -0,0 +1,1102 @@
+var/list/hallucinated_phrases = file2list("config/hallucination/hallucinated_phrases.txt")
+var/list/hallucinated_actions = file2list("config/hallucination/hallucinated_actions.txt")	//important note when adding to this file: "you" will always be replaced by the hallucinator's name
+var/list/hallucinated_thoughts = file2list("config/hallucination/hallucinated_thoughts.txt")
+
+
+/mob/living/carbon/var/next_hallucination = 0			//Hallucination spam limit var
```

Comment:
I don't quite like the declaration here. Should be like this:
````
/mob/living/carbon
    var/next_hallucination = 0 //yada
    var/list/hallucinations = list() //yada2
```
If you do this, people can find the variables by searching `carbon$` on VSC with regex.

Reply:
Also, just a thought, maybe this stuff should be human only and located in the brain, but I don't mind anyway.